clients/drcachesim/common/options.cpp

```diff
@@ -505,6 +505,14 @@ droption_t<bytesize_t>
                  "of being simulated.  This skipping may be slow for large skip values; "
                  "consider -skip_instrs for a faster method of skipping.");
 
+droption_t<bytesize_t> op_L0_filter_until_instrs(
+    DROPTION_SCOPE_CLIENT, "L0_filter_until_instrs", 0,
+    "Number of instructions for warmup trace",
+    "Specifies the number of instructions to run in warmup mode. In warmup mode, we "
+    "filter accesses through the L0{D,I}_filter caches. The size of these can be "
+    "specified using L0{D,I}_size. The filter instructions come after the "
+    "-trace_after_instrs count and before the instruction trace. ");
```

Comment:
Please add some content to drcachesim.dox.in as well about the new feature

Reply:
Which sub-section should I add the comments to? The only references to filtered traces I found were in the '
Core Simulation Support' section. Should I add the comments here?

Reply:
sec_drcachesim_partial about windows should mention the filtered-in-every-window.

How about a new section on filtered traces in general (probably should have already been there) also.

Reply:
Can we do this as a separate PR?

Reply:
I suppose, though generally tests and associated docs go in the same PR as the code changes, not just to tie them together but to ensure they are present as despite the best intentions it is not uncommon for all of us for follow-ups of any kind to get delayed and then never happen.

Reply:
Added documentation.